includes/sanitizers/class-amp-tag-and-attribute-sanitizer.php

```diff
@@ -1098,6 +1107,216 @@ private function sanitize_disallowed_attribute_values_in_node( DOMElement $node,
 		return $attributes_pending_removal;
 	}
 
+	/**
+	 * Validates the layout attributes for the given tag. This involves checking the
+	 * layout, width, height, sizes attributes with AMP specific logic.
+	 *
+	 * @version 1911070201440
+	 * @link https://github.com/ampproject/amphtml/blob/1911070201440/validator/engine/validator.js#L3937
+	 *
+	 * Adapted from the <code>validateLayout</code> method found in validator.js of the
```

Comment:
I don't think we have HTML markup in the docblocks anywhere else, have we? I'd suggest sticking with backticks instead of `<code></code>` and remove the active link, as there's a `@link` annotation directly underneath.

Reply:
A quick search of the codebase yielded no such cases, so that's good. Removed the markup in  https://github.com/ampproject/amp-wp/pull/3728/commits/08857d9f5bfc314a172a9be98c2f29846d644c96.